src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -406,6 +407,17 @@ private void AddNameAndTypeArgumentsOrParameters(INamedTypeSymbol symbol)
             }
         }
 
+        private void AddUnderlyingEnumType(INamedTypeSymbol symbol)
+        {
+            if (symbol.TypeKind == TypeKind.Enum && symbol.EnumUnderlyingType.SpecialType != SpecialType.System_Int32)
+            {
+                AddSpace();
+                AddPunctuation(SyntaxKind.ColonToken);
+                AddSpace();
+                AddSpecialTypeKeyword(symbol.EnumUnderlyingType);
```

Comment:
Maybe better like so?
```suggestion
                VisitNamedType(symbol.EnumUnderlyingType);
```